### ISSUE TITLE
create script for crontab tasks

### ISSUE THREAD

**Issue by garlick** [flux-framework/flux-accounting#450]
Problem: currently the system administrators must maintain a flux crontab file in `/etc/flux/cron.d` for accounting, so when flux-accounting requires a change it requires sys admin intervention, and when the change is version dependent, the sys admins have to ensure configuration management can deal with the transition.

From an offline discussion, one possible solution is:
- package the crontab entry but don't install it directly into `/etc/flux/cron.d`.  Let the admins control the installation of a symlink there to enable it, since it should only be enabled on a management node that has a configured database
- move the crontab commands into a shell script that the crontab entry calls so that the commands can be updated without the need to restart flux or manually update the crontab

**Comment by ryanday36** [comment]
Two other thoughts on this:

Could this be more built into the flux accounting service such that starting that service automatically turns on the updates rather than requiring a separate thing in ansible / configuration management?
Regardless of that, if would be good if the script read an update frequency from `system/conf.d/archive.toml` or similar so that all of the db configs are in one place if the system administrators want more (or less) frequent updates to fairshare, etc.

**Comment by garlick** [comment]
Good idea.  The systemd unit file does have  [ExecStartPre](https://www.freedesktop.org/software/systemd/man/latest/systemd.service.html#ExecStartPre=) and [ExecStopPost](https://www.freedesktop.org/software/systemd/man/latest/systemd.service.html#ExecStopPost=).

ExecStartPre could run the `flux cron tab <file` command but it would need to not do so if the cron tab is already installed.

ExecStopPost would need some way to look up the cron id and stop or delete it.

So a few details to think about there but maybe doable?

**Comment by cmoussa1** [comment]
Perhaps starting with just a shell script might be a somewhat straightforward way to go (at least to start)? Is there a certain directory in the flux-accounting repo that something like this would belong in?
